sbin/build.sh

```diff
@@ -263,22 +263,33 @@ removingUnnecessaryFiles()
 signRelease()
 {
   if [ "$SIGN" ]; then
-    if [[ "$OSTYPE" == "cygwin" ]]; then
-      echo "Signing release"
-      signToolPath=${signToolPath:-"/cygdrive/c/Program Files/Microsoft SDKs/Windows/v7.1/Bin/signtool.exe"}
-      # Sign .exe files
-      FILES=$(find "${OPENJDK_REPO_TAG}" -type f -name '*.exe')
-      for f in $FILES; do
-        "$signToolPath" sign /f "$CERTIFICATE" /p "$SIGN_PASSWORD" /fd SHA256 /t http://timestamp.verisign.com/scripts/timstamp.dll "$f"
-      done
-      # Sign .dll files
-      FILES=$(find "${OPENJDK_REPO_TAG}" -type f -name '*.dll')
-      for f in $FILES; do
-        "$signToolPath" sign /f "$CERTIFICATE" /p "$SIGN_PASSWORD" /fd SHA256 /t http://timestamp.verisign.com/scripts/timstamp.dll "$f"
-      done
-    else
-      echo "Skiping code signing as it's only supported on Windows"
-    fi
+    case "$OSTYPE" in
+      "cygwin")
+        echo "Signing Windows release"
+        signToolPath=${signToolPath:-"/cygdrive/c/Program Files/Microsoft SDKs/Windows/v7.1/Bin/signtool.exe"}
+        # Sign .exe files
+        FILES=$(find "${OPENJDK_REPO_TAG}" -type f -name '*.exe')
+        for f in $FILES; do
+          "$signToolPath" sign /f "$CERTIFICATE" /p "$SIGN_PASSWORD" /fd SHA256 /t http://timestamp.verisign.com/scripts/timstamp.dll "$f"
+        done
+        # Sign .dll files
+        FILES=$(find "${OPENJDK_REPO_TAG}" -type f -name '*.dll')
+        for f in $FILES; do
+          "$signToolPath" sign /f "$CERTIFICATE" /p "$SIGN_PASSWORD" /fd SHA256 /t http://timestamp.verisign.com/scripts/timstamp.dll "$f"
+        done
+      ;;
+      "darwin"*)
+        echo "Signing OSX release"
+        # Sign all executables
+        FILES=$(find "${OPENJDK_REPO_TAG}" -perm +111 -type f)
```

Comment:
This will sign anything marked executable (e.g. python/shell scripts). is that intended?

Probably worth doing:

```bash
FILES=$(find "${OPENJDK_REPO_TAG}" -perm +111 -type f || find "${OPENJDK_REPO_TAG}" -perm /111 -type f)
```

to handle people who've `brew install`'d a GNU version of `find` (as `-perm +` has been deprecated since 2005 and long since removed).

Reply:
is it possible to use a constant for `+111` so we know what files and which types of permissions are being filtered?

Reply:
I was something like this could work:
```
CAN_EXECUTE_BUT_NOT_READ="+111"
FILES=$(find "${OPENJDK_REPO_TAG}" -perm "${CAN_EXECUTE_BUT_NOT_READ}" -type f)
```

Reply:
Hmm, I think I'd find that extra indirection less clear than a comment that specified that we're checking for the executable bit (i.e https://github.com/AdoptOpenJDK/openjdk-build/pull/165#discussion_r140487777).